website/source/api/secret/transit/index.html.md

```diff
@@ -894,6 +1023,31 @@ data.
   `/transit/hmac` function. Either this must be supplied or `signature` must be
   supplied.
 
+- `batch_input` `(array<object>: nil)` – Specifies a list of items for processing.
```

Comment:
(GH won't let me add this comment outside the diff range).  Please update the `input` docs here as you did the others.